docs/quick-ingestion-guides/tableau/setup.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Setup
+---
+# Tableau Ingestion Guide: Setup & Prerequisites
+
+In order to configure ingestion from Tableau, you'll first have to enable Tableau Metadata API and you should have a user with Site Administrator Explorer permissions.
+
+## Tableau Prerequisites
+
+1. **Allow Site Administrator Explorer permissions:**
```

Comment:
Grant `Site Administrator Explorer` permissions to a user

Reply:
done